src/model/bioprotocol.v1.yaml

```diff
@@ -0,0 +1,21 @@
+$schema: http://json-schema.org/draft-04/schema#
+title: BioProtocol
+type: array
+items:
+    type: object
+    properties:
+        sequencing_number:
+            description: "section ID, looks like 's4-1'"
+            type: string
+        title:
+            description: "section title, looks like 'Isolation of LDs and treatments'"
+            type: string
+        status:
+            description: "numeric code corresponding to the current status of a protocol"
+            type: "boolean"
```

Comment:
Numeric or boolean? If it a boolean, name needs to match (`status` doesn't work), otherwise an enum should be used.

Reply:
it was changed to boolean from an integer. I'll change the description.

Why doesn't 'status' work? is it a reserved keyword or something?

Reply:
Doesn't work as in I don't know what `status: false` means (same with `status: 1`).

Reply:
how about `has_protocol` ?

Reply:
this boolean value indicates whether a 'request protocol' link should be created or a 'view protocol' link should be created. `status` seems fine to me, `has_protocol` doesn't seem so bad either

Reply:
I'm keeping `status` and have improved the description. It's clearer now.